### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PTRSC**
Updated list of municipality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,20 +86,25 @@ Gentofte - ID: 157
...
 Rebild - ID: 840
 Ringkøbing-Skjern - ID: 760
 Roskilde - ID: 265
 Rudersdal - ID: 230
 Solrød - ID: 269
 Stevns - ID: 336
 Svendborg - ID: 479
+Sweco Demo - ID: 10210
+Sweco Test B - ID: 16161
+Sweco Test T - ID: 20161
```
Comment (AngelFreak) [inline README.md:107]: These where left out as they are useless, and give a false total of supported cities.
I would suggest removing them.

Are they added by accident, or deliberately ?

**Review by AngelFreak — COMMENTED** [review]
Thanks a lot for your contribution!!
I'm really glad that you took the time to add more supported cities.

Please see my one comment before i merge it.

File: README.md (hunk 1 above)
Comment (PTRSC) [reply README.md:107]: I just took all the id and "municipalities / names" without checking them.
I remove the 3 lines of test / demo

**Comment by PTRSC** [comment]
pull request contained demo / test ID which is not usable

**Pull request closed without merging**